index.js

```diff
@@ -1,44 +1,45 @@
-const fs = require('fs')
-const YAML = require('yaml')
-const core = require('@actions/core')
+const fs = require('fs');
+const YAML = require('yaml');
+const core = require('@actions/core');
 
-const cliConfigPath = `${process.env.HOME}/.jira.d/config.yml`
-const configPath = `${process.env.HOME}/jira/config.yml`
-const Action = require('./action')
+const cliConfigPath = `${process.env.HOME}/.jira.d/config.yml`;
+const configPath = `${process.env.HOME}/jira/config.yml`;
+const Action = require('./action');
 
 // eslint-disable-next-line import/no-dynamic-require
-const githubEvent = require(process.env.GITHUB_EVENT_PATH)
-const config = YAML.parse(fs.readFileSync(configPath, 'utf8'))
+const githubEvent = require(process.env.GITHUB_EVENT_PATH);
+const config = YAML.parse(fs.readFileSync(configPath, 'utf8'));
 
-async function exec () {
+async function exec() {
   try {
     const result = await new Action({
       githubEvent,
       argv: parseArgs(),
       config,
-    }).execute()
+    }).execute();
 
     if (result) {
-      const extendedConfig = Object.assign({}, config, result)
+      const extendedConfig = Object.assign({}, config, result);
```

Comment:
This could just be an object spread

Reply:
yeah... but idk if I want to go around changing code that I don't need to, already bad enough that my vs code setup added semi colons everywhere

Reply:
Ugh is this project in standard style or something like that?  Now I want to run down their eslint config...

Reply:
their eslint config is really terrible. it was red-underlining line endings (LF vs CRLF)